tests/sles4sap/publiccloud/cluster_add_repos.pm

```diff
@@ -24,8 +24,8 @@ sub run {
 
     while (defined(my $maintrepo = shift @repos)) {
         next if $maintrepo =~ /^\s*$/;
-        if ($maintrepo =~ /Development-Tools/) {
-            record_info("MISSING REPOS", "There are Development-Tools repos in this incident, that are not uploaded to IBSM. Later errors, if they occur, may be due to these.");
+        if ($maintrepo =~ /Development-Tools/ or $maintrepo =~ /Desktop-Applications/) {
+            record_info("MISSING REPOS", "There are $maintrepo repos in this incident, that are not uploaded to IBSM. Later errors, if they occur, may be due to these.");
```

Comment:
Using $maintrepo in the record info may not be the best option. Before it was

`There are Development-tools repos in this incident`......
But now it is
`There are http://download.suse.de/ibs/SUSE:/Maintenance:/33331/SUSE_Updates_SLE-Module-Desktop-Applications_15-SP5_x86_64 repos in this incident,`...

Let me try and think of a better way, otherwise we can live with it

Reply:
In the end, we no need to mark which ones it's. If, let the paresis in place  :-)

```
record_info("MISSING REPOS", "This incident depends on at least one repository not uploaded to the IBS mirror ( $maintrepo ). Later errors, if occur, may be due to these.");
```

Reply:
I followed Jans proposal.